apps/cli/templates/defaults/remix.config.js

```diff
@@ -0,0 +1,24 @@
+/** @type {import('@remix-run/dev').AppConfig} */
+module.exports = {
+  ignoredRouteFiles: ["**/.*"],
+  serverModuleFormat: "cjs",
+  serverDependenciesToBundle: [
+    /@webstudio-is\/(?!prisma-client)/,
+    "pretty-bytes",
+    "nanoevents",
+    "nanostores",
+    "@nanostores/react",
+    /micromark/,
+    "decode-named-character-reference",
+    "character-entities",
+    /mdast-/,
+    "unist-util-stringify-position",
```

Comment:
Most of it is not used

Reply:
I didn't do a detailed check of each dep. But some of them like `@webstudio-is/` and `nanostores` and `nano-events` etc are sub-deps of some of them. So need to do a detailed check on what 's needed. Will check once by one

Reply:
Did a check with all the packages and removed the ones that are not being used now 👍